gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/MysqlDagActionStoreTest.java

```diff
@@ -70,17 +70,6 @@ public static DagActionStore getTestDagActionStore(ITestMetastoreDatabase testDb
     return new MysqlDagActionStore(config);
   }
 
-  @Test
-  public void testAddAction() throws Exception {
-    this.mysqlDagActionStore.addJobDagAction(flowGroup, flowName, flowExecutionId, jobName, DagActionStore.DagActionType.KILL);
-    //Should not be able to add KILL again when previous one exist
-    Assert.expectThrows(IOException.class,
-        () -> this.mysqlDagActionStore.addJobDagAction(flowGroup, flowName, flowExecutionId, jobName, DagActionStore.DagActionType.KILL));
-    //Should be able to add a RESUME action for same execution as well as KILL for another execution of the flow
-    this.mysqlDagActionStore.addJobDagAction(flowGroup, flowName, flowExecutionId, jobName, DagActionStore.DagActionType.RESUME);
-    this.mysqlDagActionStore.addJobDagAction(flowGroup, flowName, flowExecutionId_2, jobName, DagActionStore.DagActionType.KILL);
-  }
-
```

Comment:
why get rid of this?  I didn't recall a code change to motivate abandoning this...

Reply:
it needed to be removed when in earlier commit when i was changing the behavior of dag action store to accept duplicate entries and not throw exception. adding it back